unit-tests/run-unit-tests.py

```diff
@@ -94,7 +94,8 @@ def run(cmd, errorstatus=256, stdout=subprocess.PIPE):
         rv = subprocess.run( cmd,
                              stdout = stdout,
                              stderr = subprocess.STDOUT,
-                             universal_newlines = True )
+                             universal_newlines = True,
+                             check = True)
```

Comment:
`If check is true, and the process exits with a non-zero exit code, a CalledProcessError exception will be raised`

Is that the intent? Because in the next line you check the return code...

Reply:
I removes all references to the status and errorstatus variables as discussed